plugins/modules/nsxt_transport_nodes.py

```diff
@@ -514,10 +522,6 @@
                       'present' is used to create or update resource. 
                       'absent' is used to delete resource."
         required: true
-    transport_zone_endpoints:
```

Comment:
Bugs Fixed - 
2619445 - nsxt_transport_nodes modules needs updates based on new spec
2648943 - updation of only tags is not supported by 'nsxt_transport_nodes' module